addons/sourcemod/scripting/szf/config.sp

```diff
@@ -12,10 +13,12 @@ enum struct ConfigMelee
 
 ConfigMelee g_ConfigMeleeDefault;
 ArrayList g_aConfigMelee;
+StringMap g_OrigWeaponIndexes;
 
 void Config_Init()
 {
 	g_aConfigMelee = new ArrayList(sizeof(ConfigMelee));
+	g_OrigWeaponIndexes = Config_LoadWeaponIndexes();
```

Comment:
Should be done in `Config_Refresh` to allow map-change or `sm_szfreload` refresh it.